lib/routemap.c

```diff
@@ -2867,6 +2923,30 @@ DEFUN (no_rmap_description,
 	return CMD_SUCCESS;
 }
 
+DEFUN (debug_rmap,
+       debug_rmap_cmd,
+       "debug route-map",
+       DEBUG_STR
+       "Debug option set for route-maps\n")
+{
+	rmap_debug = true;
+	return CMD_SUCCESS;
+}
+
+DEFUN (no_debug_rmap,
+       no_debug_rmap_cmd,
+       "no debug route-map",
```

Comment:
Could you please collapse this DEFUN into the previous one?
```
DEFUN (debug_rmap,
       debug_rmap_cmd,
       "[no] debug route-map",
       NO_STR
       DEBUG_STR
       "Debug option set for route-maps\n")
{
	rmap_debug = !no;
	return CMD_SUCCESS;
}
```

Reply:
This gives following error.

lib/routemap.c: In function 'debug_rmap':
lib/routemap.c:3071:16: error: 'no' undeclared (first use in this function)
  rmap_debug = !no;
                ^
lib/routemap.c:3071:16: note: each undeclared identifier is reported only once f

Is there any other way of doing this?

for all other debug commands, [no] is handled in a  separate command.

Reply:
sorry, should have wrote `DEFPY` not `DEFUN`